coalib/__init__.py

```diff
@@ -7,10 +7,46 @@
 
 import sys
 from os.path import join, dirname
+from subprocess import check_output, CalledProcessError
 
 
 VERSION_FILE = join(dirname(__file__), 'VERSION')
 
+COMMIT_FILE = join(dirname(__file__), 'COMMIT')
+
+
+def get_current_commit():
```

Comment:
this needs to be run in the coalib directory, otherwise it will be wrong.
It is more versatile if the caller specifies the directory that it should be run in, as an argument to this function.